source/_components/heos.markdown

```diff
@@ -10,13 +10,18 @@ footer: true
 logo: heos.png
 ha_category: Media Player
 ha_release: 0.92
+ha_iot_class: Local Push
+redirect_from:
```

Comment:
Only for existing components

Reply:
Oh... we added all of these to SmartThings which wasn't existing. :)  Thought it was to preserve folks ability to key in the URL in the old syntax.  I can remove.